packages/shared/.npmignore

```diff
@@ -0,0 +1 @@
+# empty npmignore to ensure all required files (e.g., in the dist folder) are published by npm
```

Comment:
This should contain all directories/files we don't want to be published to npm. Like `src`, 'node_modules', `.eslintrc`, etc.